### ISSUE TITLE
PyTorch should be required

### ISSUE THREAD

**Issue by ted537** [Vennify-Inc/chatting-transformer#14]
This error occurs on a fresh venv install

```
>>> from chattingtransformer import ChattingGPT2
Neither PyTorch nor TensorFlow >= 2.0 have been found.Models won't be available and only tokenizers, configurationand file/data utilities can be used.
>>> gpt2 = ChattingGPT2()
10/22/2020 20:09:02 - INFO - chattingtransformer.chatting_transformer -   Loading "gpt2"...
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/ted/Documents/git/chatting-transformer/tmp/venv/lib/python3.7/site-packages/chattingtransformer/chatting_transformer.py", line 52, in __init__
    pad_token_id=pad_token_id)
  File "/home/ted/Documents/git/chatting-transformer/tmp/venv/lib/python3.7/site-packages/transformers/utils/dummy_pt_objects.py", line 221, in from_pretrained
    requires_pytorch(self)
  File "/home/ted/Documents/git/chatting-transformer/tmp/venv/lib/python3.7/site-packages/transformers/file_utils.py", line 380, in requires_pytorch
    raise ImportError(PYTORCH_IMPORT_ERROR.format(name))
ImportError: 
AutoModelForCausalLM requires the PyTorch library but it was not found in your enviromnent. Checkout the instructions on the
installation page: https://pytorch.org/get-started/locally/ and follow the ones that match your enviromnent.
```

**Comment by EricFillion** [comment]
Nice catch! The command "pip installing torch" fixes it. I added torch to the requirments in #15

**Comment by EricFillion** [comment]
Resolved in version 1.0.1.
